mlcube/vision_data_selection/mlcube.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) MLCommons and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""MLCube handler file"""
+import os
+import subprocess
+
+import typer
+import yaml
+from selection import Predictor
+
+
+typer_app = typer.Typer()
+
+
+class DownloadTask:
+    """Download samples and eval data"""
+
+    @staticmethod
+    def run(parameters_file: str, output_path: str) -> None:
+
+        cmd = "python3 download_data.py"
+        cmd += f" --parameters_file={parameters_file} --output_path={output_path}"
+        splitted_cmd = cmd.split()
+
+        process = subprocess.Popen(splitted_cmd, cwd=".")
+        process.wait()
+
+
+class SelectTask:
+    """Run select algorithm"""
+
+    @staticmethod
+    def run(
+        input_path: str, embeddings_path: str, parameters_file: str, output_path: str
+    ) -> None:
+
+        with open(parameters_file) as f:
+            params = yaml.full_load(f)
+
+        print("Creating predictor")
+        predictor = Predictor(embeddings_path)
+
+        input_files = [
```

Comment:
This limits the amount of classes the mlcube works with. We should change this to use the yaml, so if in the future we add more classes the pipeline still functions properly